src/main/java/sirius/biz/storage/layer2/URLBuilder.java

```diff
@@ -382,6 +384,15 @@ public String safeBuildURL(String fallbackUri) {
         return buildURL().orElseGet(() -> createBaseURL().append(fallbackUri).toString());
     }
 
+    /**
+     * Builds the URL from the fallbackUri.
+     *
+     * @return the URL with the fallbackUri
+     */
+    public String buildFallbackURL() {
```

Comment:
shouldn't this be optional in case the fallback uri is null? or should we simply make createBaseURL public?